### PR TITLE
chore: konflux: m6i.2xlarge for smaller clusters

### PR DESCRIPTION
**Pull request by psturc**
https://redhat-internal.slack.com/archives/C02FANRBZQD/p1724758372368729?thread_ts=1724326956.199439&cid=C02FANRBZQD

### CODE REVIEW

File: clusters/hosted-mgmt/hive/pools/rhtap/rhtap-aws-us-west-2.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ stringData:
       name: worker
       platform:
         aws:
-          type: m5.xlarge
-      replicas: 4
+          type: m6i.2xlarge
```
Comment (psturc) [inline clusters/hosted-mgmt/hive/pools/rhtap/rhtap-aws-us-west-2.yaml:15]: https://instances.vantage.sh/aws/ec2/m5.xlarge vs. https://instances.vantage.sh/aws/ec2/m6i.2xlarge

Comment (dheerajodha) [reply clusters/hosted-mgmt/hive/pools/rhtap/rhtap-aws-us-west-2.yaml:15]: wdyt about https://instances.vantage.sh/aws/ec2/m5.2xlarge?

Comment (dheerajodha) [reply clusters/hosted-mgmt/hive/pools/rhtap/rhtap-aws-us-west-2.yaml:15]: It also has 8 vcpus 32 GB ram per node

**Pull request merged**